ql/src/test/results/clientpositive/alter_rename_table.q.out

```diff
@@ -131,7 +131,7 @@ STAGE PLANS:
   Stage: Stage-0
     Rename Table
       table name: source.src
```

Comment:
the old tablename doesn't qualified hat the "category"
we should be consistent in showing or not showing it category